main.go

```diff
@@ -67,8 +68,46 @@ func usage() {
 }
 
 func main() {
+	log.SetFlags(0)
+	log.SetPrefix("")
+
 	flag.Usage = usage
 	flag.Parse()
+
+	// do some basic validation on the outfile format
+	// indexing is safe because we're searching for an ascii char
```

Comment:
> The general case of checking fmt strings is hard.

Completely agree. I think I've erroneously gone for optimisation over readability here... I'll push up a simpler check.